pkgs/by-name/pl/playit/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "playit";
+  version = "0.15.13";
+
+  src = fetchFromGitHub {
+    owner = "playit-cloud";
+    repo = "playit-agent";
+    rev = "v${version}";
+    hash = "sha256-RRN0KAgFVXQGU6LdNWClBFlqO+Nl4SMNXDWfV0lOhAE=";
+  };
+
+  cargoHash = "sha256-PMaDU/6Y/t78uJ98qKGc+4WDX18SPN8PwLZqOqYW4yo=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+  ];
+
+  doCheck = false;
```

Reason?

---

nix-unit thought it was neccses

---

`nix-init`? You should test it yourself

---

It worked, I do not have a darwin machine to test on.

---

ofBorg will run the builds here. You can see if it fails

---

Still, please either remove it or write reason in comment:
```suggestion
  doCheck = false; # Whatever reason
```

---

Apparently, because `doCheck` required internet access?

https://github.com/pedorich-n/playit-nixos-module/blob/337678d5c5c828c6816c70e2386d3a113766d657/nix/package.nix#L18-L19